.prettierignore

```diff
@@ -0,0 +1,2 @@
+# Ignore artifacts:
+tests/data/*
```

Comment:
```suggestion
tests/data/*/**
```
This will keep changes to `tests/data/README.md` but skip all the sub dirs. This is also what we use in `.gitattributes`.

P.S. You still need to undo changes to these files in this PR.